Sources/JSONMatcher/Builder.swift

```diff
@@ -14,13 +14,13 @@ struct Builder {
         }
     }
 
-    private func buildRawJSONElement<T>(object: T) -> BaseElementType {
+    fileprivate func buildRawJSONElement<T>(_ object: T) -> BaseElementType {
```

Comment:
I guess so, I will correct this, and try to "privatize" all other functions and see how far I can go with that without breaking the code.